awx/main/tests/functional/test_credential.py

```diff
@@ -96,11 +99,12 @@ def test_default_cred_types():
         'satellite6',
         'scm',
         'ssh',
-        'thycotic_dsv',
+        #        'thycotic_dsv',
```

Comment:
I had to comment out this line as well for the test to pass. Any pointers as to why this might be causing the issue ?